### PR TITLE
Added side drawer for the Gmail UI

### PR DESCRIPTION
**Pull request by blacpythoz**
Added side bar drawer UI and drawer open on navigation icons.

### CODE REVIEW

File: app/src/main/java/com/guru/composecookbook/ui/demoui/gmail/GmailDrawer.kt [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    Row {
+        Icon(asset = icon, modifier = Modifier.padding(16.dp))
+        Text(
+            modifier = Modifier.weight(1f)
+                .align(Alignment.CenterVertically)
+                .padding(start=8.dp),
+            text = title,
+            fontFamily = FontFamily.SansSerif,
+            fontWeight = FontWeight.SemiBold,
+            fontSize = 14.sp,
```
Comment (Gurupreet) [inline app/src/main/java/com/guru/composecookbook/ui/demoui/gmail/GmailDrawer.kt:84]: We can always use existing typography.

File: app/src/main/java/com/guru/composecookbook/ui/demoui/gmail/GmailHome.kt [hunk 2]
```diff
@@ -26,6 +26,9 @@ import com.guru.composecookbook.ui.Animations.FloatMultiStateAnimationExplode
 
 @Composable
 fun GmailHome() {
+
```
Comment (Gurupreet) [inline app/src/main/java/com/guru/composecookbook/ui/demoui/gmail/GmailHome.kt:29]: No need for first space 👍🏽

**Review by Gurupreet — APPROVED** [review]
Great work thanks fo adding needed stuff 👍🏽

**Comment by blacpythoz** [comment]
yeah, I will add more stuff to it.

**Comment by gur308** [comment]
Great man keep it up :) Create an issue so we don't have any duplicate work going on.

**Comment by Gurupreet** [comment]
Great keep it up 👍🏽 Creating an issue before starting would be great to avoid duplicate work 💯

**Pull request merged**